util/csv.js

```diff
@@ -0,0 +1,85 @@
+export class CoordinapeCSVParseError extends Error {
```

Comment:
it seems that this is related with Coordinape CSV parsing but filename is generic to `csv.js`

Reply:
I thought there might be other csv upload cases, and there may not be. I’ll circle back on this in a new PR @edderrd— thanks!